### ISSUE TITLE
Future twiggy conflicts

### ISSUE THREAD

**Issue by sodapopcan** [kristijanhusak/vim-create-pr#2]
I have some major refactoring for twiggy.vim on the horizon.  One issue is that I no longer want to use `^` and `V` for pushing and pulling.  I was always a fan of `^` until I started working at a place that uses really long branch names—I got my co-worker to try out twiggy and he hit `$` to read the rest of the branch name then `^` to get back but it pushed!   I also don't like to use mappings that still make sense in unmodifiable buffers (`^` and `V` are the only two that violate this).

TL;DR, I am soon going to be deprecating those mappings in favour of `P` and `p`.

To keep in line with twiggy as well, I really like to have just one character mappings.  I was trying to think of a good one for opening a PR.  The best I can come up with is `X` (as it stands, lowercase is for local stuff and uppercase is for remote stuff).

I'm also softening to the idea of just including this in twiggy if you're still open to it.

**Comment by kristijanhusak** [comment]
Yeah that sounds ok. I didn't knew what to use back then, and `pr` looked ok.

I'm open for moving it there. You can easily put it in a separate autoload file and don't worry about it until it's used.
